### ISSUE TITLE
Resolve current accessibility issues

### ISSUE THREAD

**Issue by Snuggle** [Snuggle/snugg.ie#23]
A number of accessibility issues were found by installing the Netlify a11y plugin #22, these should all be fixed ASAP!

Once these have been resolved, future additions of any accessibility issues will fail the CI/CD build.

https://app.netlify.com/teams/snuggle/builds/60a6ef318743cf0007ab9d11

**Comment by Snuggle** [comment]
Resolved by 5199b787a8536c2c20eceb129abc8631129cebbc, 95d68bbf10fec0099ec8b91ef89b3d4ee3ff0af7 and 468da5234cbd349e3d690f2bdeba4b8a92a07e0f. 😊
